files/scponly-chroot.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+ALTROOT="$1"; shift
+BINARIES="$@"
+
+for b in $BINARIES
+do
+    d=$(dirname "${b}")
+    if [ ! -d "${ALTROOT}${d}" ]; then
+        /bin/mkdir -p "${ALTROOT}${d}"
+    fi
+    /bin/cp "${b}" "${ALTROOT}${d}"
+
+    LIBS=$(ldd "${b}" | awk '{ print $3 }' | egrep -v -v ^'\(')
+
+    for i in ${LIBS}; do
+        d=$(dirname "${i}")
+        if [ ! -d "${ALTROOT}${d}" ]; then
+            /bin/mkdir -p "${ALTROOT}${d}"
+        fi
+            /bin/cp "${i}" "${ALTROOT}${d}"
+    done
+
+    ld=$(ldd "${b}" | grep 'ld-linux' | awk '{ print $1 }')
+    d=$(dirname "${ld}")
+
+    if [ ! -d "${ALTROOT}${d}" ]; then
+        /bin/mkdir -p "${ALTROOT}${d}"
+    fi
+    /bin/cp "${ld}" "${ALTROOT}${d}"
+
+done
+
+/bin/cp "/lib64/libnss_* ${ALTROOT}/lib64/"
```

Comment:
This is likely what's causing your issue
```suggestion
/bin/cp /lib64/libnss_* ${ALTROOT}/lib64/
```

Reply:
If I omit the double quotes whatever linter I have is saying to double quote to prevent globbing or word splitting. Should I be double quoting somehow else?

Reply:
I think you need it like this instead:
```bash
/bin/cp "/lib64/libnss_*" "${ALTROOT}/lib64/"
```
However you should double check that works.